mypy/fastparse.py

```diff
@@ -249,6 +249,8 @@ def __init__(self,
                  is_stub: bool,
                  errors: Errors) -> None:
         self.class_nesting = 0
```

Comment:
Shouldn't you delete this variable? It's no longer updated.

Reply:
Yes, I forgot it.